release-3.7.0-announcement.md

```diff
@@ -7,7 +7,7 @@ This release re-organizes Server lists to make room for more servers:
 - Choral/Barbershop has been added.
 - Classical/Folk/Choral has been renamed to Classical/Folk.
 
-Besides that, tens of usability improvements, enhancements, bugfixes and optimizations have been integrated.
+Besides that, numerous of usability improvements, enhancements, bugfixes and optimizations have been integrated.
```

Comment:
I think you don’t need the preposition of here.

Reply:
Meh, still too early. Thanks. :)